pkg/asset/tls/certkey.go

```diff
@@ -0,0 +1,171 @@
+package tls
```

Comment:
> Golint comments: should have a package comment, unless it's in another file for this package.

I think this is just a limitation of the `golint` plugin, because you have package docs in a sibling file:

```console
$ git grep '// Package tls ' 0a7e6f4 -- pkg/asset/tls
0a7e6f4:pkg/asset/tls/doc.go:// Package tls defines and generates the tls assets based on its dependencies.
```

And the `golint` Prow job [is happy][1], so I don't think we need to worry about these.

[1]: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_installer/145/ci-pull-openshift-installer-golint/219/